### ISSUE TITLE
Extend support for GPUs

### ISSUE THREAD

**Issue by dignifiedquire** [filecoin-project/bellman#24]
Current list of requested GPUs

- [ ] Tesla P100
- [ ] Tesla V100
- [ ] 1650 super
- [ ] 1650
- [ ] 1050ti

In addition we should add an environment flag `BELLMAN_SUPPORT_GPUS` which allows specifying additional cards, without having to recompile

```
> BELLMAN_SUPPORT_GPUS="<card name>:<cuda cores>,<card name>:<cuda cores>
```

**Comment by Dengminer** [comment]
Tesla T4
Tesla P4

**Comment by lanzafame** [comment]
Titan RTX as well please

**Comment by porcuquine** [comment]
@keyvank Is this closed by #27? If so, please close. (I think you tried to have this happen automatically. You can't just include the issue number. You have to write something like:
```
closes #24.
```
